presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -136,21 +143,28 @@
     private static final String WILDCARD_EXPRESSION = "";
     private static final int BATCH_GET_PARTITION_MAX_PAGE_SIZE = 1000;
     private static final int BATCH_CREATE_PARTITION_MAX_PAGE_SIZE = 100;
+    private static final Comparator<Partition> PARTITION_COMPARATOR =
+            Comparator.comparing(p -> p.getValues(), Comparators.lexicographical(String.CASE_INSENSITIVE_ORDER));
```

Comment:
Nit: use a method reference and static import
```java
private static final Comparator<Partition> PARTITION_COMPARATOR =
        comparing(Partition::getValues, lexicographical(CASE_INSENSITIVE_ORDER));
```

Reply:
Changing.